clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/MetadataRepoBasedStaleTopicCheckerImpl.java

```diff
@@ -28,8 +28,8 @@ public boolean shouldTopicExist(String topic) {
           return false;
         }
       } else if (Version.isRealTimeTopic(topic)) {
-        if (!store.isHybrid()) {
-          LOGGER.warn("Store {} is not hybrid currently, but found real-time topic {}", storeName, topic);
+        if (!Version.containsHybridVersion(store.getVersions())) {
+          LOGGER.warn("Store {} does not have any hybrid version for real-time topic: {}", storeName, topic);
```

Comment:
should this be a warn?

Reply:
I dont get the question. Which log tag do you prefer?

Reply:
nvm, I didnt see we are printing RT topic name as well.